actors/common/src/main/java/org/sunbird/learner/actors/CourseBatchManagementActor.java

```diff
@@ -716,9 +716,9 @@ private void updateCourseBatch(Request actorMessage) {
   }
 
   private void validateUpdateBatchStartDate(Date startDate) {
-    if (startDate != null) {
+    SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
Why is there a change in CourseBatchManagementActor when the issue is with regard to user bulk upload?